src/textlint-core.js

```diff
@@ -41,8 +45,22 @@ export default class TextlintCore {
     }
 
     /**
-     * Register rules to EventEmitter.
-     * if want to release rules, please call {@link this.resetRules}.
+     * register Processors
+     * @param {Object} processors
+     */
+    setupProcessors(processors = {}) {
```

Comment:
Currently, processor have not options.
I think that processor have a options in the fureture, and pass it as 2nd argumetns.